WordPressKit/CommentServiceRemoteXMLRPC.m

```diff
@@ -119,11 +119,20 @@ - (void)updateComment:(RemoteComment *)comment
 {
     NSParameterAssert(comment.commentID != nil);
     NSNumber *commentID = comment.commentID;
-    NSArray *extraParameters = @[
-                                 comment.commentID,
-                                 @{@"content": comment.content},
-                                 ];
+    
+    NSDictionary *commentDictionary = @{
+        @"content": comment.content,
+        @"author": comment.author,
+        @"author_email": comment.authorEmail,
+        @"author_url": comment.authorUrl,
+    };
+    
+    NSArray *extraParameters = @[comment.commentID,
+                                 commentDictionary,
+    ];
```

Comment:
Just one super minor, optional suggestion on formatting the array:
```suggestion
    NSArray *extraParameters = @[
                                 comment.commentID,
                                 commentDictionary,
                                 ];
```
I think the recent Xcode no longer provides spacing for multi-line array declarations this way anymore. 😅